experiments/orbital_discriminability/lugre_snapshot_discriminability.py

```
@@ -0,0 +1,1058 @@
+        separators=None if pretty else (",", ":"),
+        sort_keys=True,
+    )
+
+
+def canonical_sha256(path: Path) -> str:
+    return sha256(Path(path).read_bytes().replace(b"\r\n", b"\n")).hexdigest()
+
+
+def _git_commit() -> str:
+    return subprocess.check_output(
+        ["git", "rev-parse", "HEAD"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)